api/src/main/java/org/openmrs/module/hivtestingservices/util/MedicDataExchange.java

```
@@ -175,13 +183,16 @@ private ObjectNode processRegistrationPayload (ObjectNode jNode) {
         tmp.put("tmp.birthdate_type","age");
         tmp.put("tmp.age_in_years", jsonNode.get("patient_ageYears") != null ? jsonNode.get("patient_ageYears").getTextValue() : "");
         discriminator.put("discriminator","json-registration");
+        String creator =   jsonNode.path("meta").path("created_by") != null ? jsonNode.path("meta").path("created_by").getTextValue() : "";
+        String providerId = checkProviderNameExists(creator);
+        String systemId = confirmUserNameExists(creator);
 
         encounter.put("encounter.location_id",locationId != null ? locationId.toString() : "1");
```

Instead of defaulting to location  1, it is better we just throw an error.